web/lib/models/event-emails.js

```diff
@@ -0,0 +1,75 @@
+const Email = require('./email');
+
+const port = process.env.ZEN_PORT || 8000;
+const host = process.env.ZEN_HOSTNAME || 'localhost';
+const protocol = process.env.ZEN_PROTOCOL || 'http';
+
+class EventEmail extends Email {
+  formatEventDate(event) {
+    if (event.type === 'recurring') {
+      return `${this.formatDate(event.dates[0].startTime)} - ${this.formatDate(event.dates[event.dates.length - 1].startTime)}`;
```

Comment:
Not a fan of this... Suggests it's on every day instead of every week/two weeks/month. Should probably be the same as frontend..?

Reply:
nuxt?